java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerService.java

```diff
@@ -29,4 +31,11 @@ public void createNewCustomer(){
             System.out.println("A new row has been inserted!!!");
         }
     }
+// -----------GETTING ALL CUSTOMER (DATABASE)------------ //
+    public List<Customer> gettingAllCustomers(){
+        String sql = "SELECT * FROM customer";
+        //query means searching for something
+        List<Customer> customersList = jdbcTemplate.query(sql, new BeanPropertyRowMapper<>(Customer.class));
+        return  customersList;
+    }
```

Comment:
do we need these comments?

Reply:
these comments helps me to read my code. So i am going to stay with them.